src/index.js

```diff
@@ -1,25 +1,28 @@
 // eslint-disable-next-line no-unused-vars
 import _ from 'lodash';
```

Comment:
- [ ] Kindly remove this Eslint disabling comment. `Eslint` helps us to write clean and industry-standard codes. To fix this lint error, kindly remove this line `import _ from 'lodash';`. You are not using any `lodash` function for this project. 👍